src/libraries/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -3,7 +3,7 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Security.Cryptography.X509Certificates</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <NoWarn>$(NoWarn);CS3016;CA5379;CA5384;CA5385</NoWarn>
+    <NoWarn>$(NoWarn);CA5384</NoWarn>
```

Comment:
@bartonjs, do you know why so many codes were NoWarn'd?  I'm wondering if maybe there's some analysis that runs as not part of the repo somewhere?  Or maybe these are just stale?

Reply:
Two of the ones you're removing now were added by you a couple of months ago (https://github.com/dotnet/corefx/pull/39703/files#diff-6c974adbac02a90ca50c70e3a3446944R7) :smile:.

Perhaps we've changed some analyzer setting defaults, or had some false positives that the analyzers no longer complain about.

Reply:
Heh, thanks :-)